config/tool_destinations.yml.sample

```diff
@@ -121,7 +121,9 @@
 #         destination: fail
 #         fail_message: Don't do that
 #         arguments:
-#           careful: true
+#           my_section:
```

Comment:
@lecorguille can you add a slightly more complex example, or more real world like?